sw/ground_segment/python/gvf/gvfFormation.py

```diff
@@ -53,13 +53,16 @@ def message_recv(ac_id, msg):
     return
 
 def formation(B, ds, radius, k):
-    no_telemetry = 0
     for ac in list_aircraft:
-        if ac.a == -999 or ac.XY[0] == -999:
-            print "Waiting for telemetry of aircraft ", ac.id
-            no_telemetry = 1
-
-    if no_telemetry:
+        waiting_for_msgs = 0
+        if ac.a == -999:
+            print "Waiting for GVF msg of aircraft ", ac.id
```

Comment:
please add `from __future__ import print_function` at the beginning and use the print function, so this also works with python3